portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Ana-Maria's Gallery</title>
+    <link rel="stylesheet" href="galleryStyle.css">
+    <script src="galleryScript.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Postcards from Wien</h1>
+      <p>I added those photos for us to remember about the good times
+          when we could travel, so here are some pics from last summer :)</p>
+      
+      <div class="smallGallery">
+
+        <div class="row">
+          <div class="column">
+            <img src="images/1.jpeg" style="width:100%" 
+            onclick="openGallery();showCurrentSlide(1)" class="hover-shadow cursor">
```

Comment:
You are using the `style` attribute and the `class` attribute at the same time here. This means there are two places where styling is defined (i.e. here in the HTML and also in the CSS too) which is a bit annoying since you might need to change two files.

Generally speaking it is worth putting anything in the `style` attribute into the CSS classes so that it is all defined together in one place.

So you could add `width:100%` into `hover-shadow`, `cursor` or add a new class for it if those don't make sense. An alternative would be to just do this using selectors in the CSS to pick only the images that are within a `row` and `column`, e.g. `.row > .column > img {width: 100%;}`

Reply:
Alright! I will keep that in mind :)